data/frontier jobs.txt

```diff
@@ -0,0 +1,174 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Frontier delivery [0]"
+	name "Frontier delivery to <planet>"
+	job
+	repeat
+	description `Deliver <cargo> to the frontier world of <destination>. Frontier worlds have a high rate of piracy, so a pirate prescense is expected. Payment is <payment>.`
+	cargo random 5 2 .1
+	to offer
+		random < 60
+	source
+		not attributes "frontier"
+	destination
+		distance 2 10
+		attributes "frontier"
+	npc
+		government "Pirate"
+		personality nemesis harvests plunders
+			confusion 30
+		system destination
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+			confusion 30
+		system destination
+		fleet "Small Northern Pirates" 2
+	on complete
+		payment 4000 180
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+
+mission "Frontier delivery [1]"
+	name "Frontier delivery to <planet>"
+	job
+	repeat
+	description `Deliver <cargo> to the frontier world of <destination>. Frontier worlds have a high rate of piracy, so a pirate prescense is expected. Payment is <payment>.`
+	cargo random 25 2 .05
+	to offer
+		random < 40
+	source
+		not attributes "frontier"
+	destination
+		distance 2 10
+		attributes "frontier"
+	npc
+		government "Pirate"
+		personality nemesis harvests plunders
+			confusion 20
+		system destination
+		fleet "Small Northern Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+			confusion 20
+		system destination
+		fleet "Small Northern Pirates" 3
+	on complete
+		payment 4000 180
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+
+mission "Frontier delivery [2]"
+	name "Frontier delivery to <planet>"
+	job
+	repeat
+	description `Deliver <cargo> to the frontier world of <destination>. Frontier worlds have a high rate of piracy, so a pirate prescense is expected. Payment is <payment>.`
+	cargo random 40 2 .02
+	to offer
+		random < 20
+	source
+		not attributes "frontier"
+	destination
+		distance 2 10
+		attributes "frontier"
+	npc
+		government "Pirate"
+		personality nemesis harvests plunders
+		system destination
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+	on complete
+		payment 4000 180
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+
+mission "Frontier Emigration [0]"
+	name "Emigration to <planet>"
+	job
+	repeat
+	description "These <bunks> passengers have finally saved up enough money to move to a more prosperous world. Bring them to <destination> where they will start their new life. Payment is <payment>."
```

Comment:
It might be better to rewrite this into something more concise:
`These <bunks> passengers have finally saved up enough money to start a new life on <destination>, and are willing to pay you <payment> to transport them.`

Maybe the `up` in `saved up enough` can be taken out.